core/src/main/java/org/mskcc/cbio/portal/dao/MySQLbulkLoader.java

```diff
@@ -244,7 +244,8 @@ private int loadDataFromTempFileIntoDBMS() throws DaoException, IOException {
         	 if (stmt.getWarnings() != null) {
         		 otherDetails = "More error/warning details: " + stmt.getWarnings().getMessage();
              }
-             throw new DaoException("DB Error: only "+updateCount+" of the "+nLines+" records were inserted in `" + tableName + "`. " + otherDetails);
+             throw new DaoException("DB Error: only "+updateCount+" of the "+nLines+" records were inserted in `" + tableName + "`. " + otherDetails + 
+            		 " See tmp file for more details: " + tempFileName);
```

Comment:
yep, can be quite handy. I believe I used this to troubleshoot one of the unit tests that was failing after the refactoring.